api/Navigator.json

```diff
@@ -2382,6 +2382,54 @@
           }
         }
       },
+      "serial": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/serviceWorker",
```

Comment:
This URL and that in `WorkerNavigator` needs to be updated.

Reply:
Fixed. PTAL